<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formpwd</class>
 <widget class="QWidget" name="Formpwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="pwdstackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>忘记密码-修改密码</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>手机号/用户名/邮箱/卡号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 登录按钮基础样式（橙色边框+橙色背景，搭配白色文字） */
QPushButton {
    /* 橙色边框：#ff7d00 是常用登录页橙色，可替换为 #ff9900（浅橙）、#ff6600（深橙） */
    border: 2px solid #ff7d00; 
    border-radius: 4px; /* 圆角（和输入框保持一致，更协调） */
    background-color: #ff7d00; /* 按钮背景色（和边框同色，一体化效果） */
    color: white; /* 文字白色 */
    font-size: 16px; /* 文字大小 */
    height: 44px; /* 按钮高度（和输入框呼应） */
}

/* 悬浮状态（边框加深+背景色变深，更有交互感） */
QPushButton:hover {
    border-color: #ff6600; /* 悬浮时边框深橙色 */
    background-color: #ff6600; /* 悬浮时背景深橙色 */
}

/* 点击状态（边框再加深+背景色压暗，模拟按压效果） */
QPushButton:pressed {
    border-color: #e65c00; /* 点击时边框更深橙色 */
    background-color: #e65c00; /* 点击时背景更深橙色 */
}

/* 禁用状态（可选：如果按钮需要禁用，边框和背景变灰色） */
QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>下一步，验证</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip1Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
     </property>
     <property name="text">
      <string>请输入用户名！</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;绑定手机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameshow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneshow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;验证码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>6位数字</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 登录按钮基础样式（橙色边框+橙色背景，搭配白色文字） */
QPushButton {
    /* 橙色边框：#ff7d00 是常用登录页橙色，可替换为 #ff9900（浅橙）、#ff6600（深橙） */
    border: 2px solid #ff7d00; 
    border-radius: 4px; /* 圆角（和输入框保持一致，更协调） */
    background-color: #ff7d00; /* 按钮背景色（和边框同色，一体化效果） */
    color: white; /* 文字白色 */
    font-size: 16px; /* 文字大小 */
    height: 44px; /* 按钮高度（和输入框呼应） */
}

/* 悬浮状态（边框加深+背景色变深，更有交互感） */
QPushButton:hover {
    border-color: #ff6600; /* 悬浮时边框深橙色 */
    background-color: #ff6600; /* 悬浮时背景深橙色 */
}

/* 点击状态（边框再加深+背景色压暗，模拟按压效果） */
QPushButton:pressed {
    border-color: #e65c00; /* 点击时边框更深橙色 */
    background-color: #e65c00; /* 点击时背景更深橙色 */
}

/* 禁用状态（可选：如果按钮需要禁用，边框和背景变灰色） */
QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>下一步，验证</string>
     </property>
    </widget>
    <widget class="ClickableLabel" name="getcodeLabel1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;发送验证码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip2Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
     </property>
     <property name="text">
      <string>请输入验证码！</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip3Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
     </property>
     <property name="text">
      <string>验证码错误！</string>
     </property>
    </widget>
    <widget class="ClickableLabel" name="backLabel_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#adadad;&quot;&gt;&amp;lt;返回上一步&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#000000;&quot;&gt;新密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请设置新密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#000000;&quot;&gt;确认新密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdEdit_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc; /* 浅灰色边框（默认可见） */
    border-radius: 4px; /* 可选：圆角更美观 */
    padding: 0 10px; /* 可选：输入内容不贴边 */
    height: 40px; /* 可选：统一高度 */
}
QLineEdit:focus {
    border-color: #0066cc; /* 聚焦时变蓝色边框（更醒目） */
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>再次输入密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 登录按钮基础样式（橙色边框+橙色背景，搭配白色文字） */
QPushButton {
    /* 橙色边框：#ff7d00 是常用登录页橙色，可替换为 #ff9900（浅橙）、#ff6600（深橙） */
    border: 2px solid #ff7d00; 
    border-radius: 4px; /* 圆角（和输入框保持一致，更协调） */
    background-color: #ff7d00; /* 按钮背景色（和边框同色，一体化效果） */
    color: white; /* 文字白色 */
    font-size: 16px; /* 文字大小 */
    height: 44px; /* 按钮高度（和输入框呼应） */
}

/* 悬浮状态（边框加深+背景色变深，更有交互感） */
QPushButton:hover {
    border-color: #ff6600; /* 悬浮时边框深橙色 */
    background-color: #ff6600; /* 悬浮时背景深橙色 */
}

/* 点击状态（边框再加深+背景色压暗，模拟按压效果） */
QPushButton:pressed {
    border-color: #e65c00; /* 点击时边框更深橙色 */
    background-color: #e65c00; /* 点击时背景更深橙色 */
}

/* 禁用状态（可选：如果按钮需要禁用，边框和背景变灰色） */
QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
    <widget class="ClickableLabel" name="backLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#adadad;&quot;&gt;&amp;lt;返回上一步&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Noto Sans SC'; font-size:12px; color:#666666;&quot;&gt;密码需为 8-20 位字母、数字和符号的组合，不含空格&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip4Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
     </property>
     <property name="text">
      <string>密码格式不正确！</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip5Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red; font-size: 12px; visibility: hidden;</string>
     </property>
     <property name="text">
      <string>两次密码输入不一致！</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="ClickableLabel" name="backLabel_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#00ffff;&quot;&gt;返回登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>on_switchLabel_clicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
